### DISCUSSION TITLE
how to use css snippet?

### DISCUSSION THREAD

**Discussion by Crownburger** [mProjectsCode/obsidian-meta-bind-plugin#95] [General]
Hi,

can someone please explain how my css snippet needs to look like to reduce the Text Input Box to 200px width instead of full width? My .md File looks like this:

```meta-bind
INPUT[text(class(Test)):tag]
```

I'm on Android.

Thanks!

**Comment by JNathan16** [comment]
Hey did you ever figure out how to reduce the text input box ? 
I think I running into a similar issue. I am using the desktop version.

**Comment by Lykhant** [comment]
I figured it out!

After looking throughout the css of the extension, it turns out that you shouldn't modify the size of the target. You need to modify the **children**. What that means is that you need to append `> *` to the class name, as in `.Test > *`.

So, if we were to make the input take all of the page's space, the css would look like this:
```css
.input-w-full > * {
  width: 100%;
}
```
Then, we apply it as normal:
```
INPUT[text(class(input-w-full)):tag]
```

Here's the result:
![image](https://github.com/mProjectsCode/obsidian-meta-bind-plugin/assets/18752330/bd76963a-3ec8-4918-96c3-310236a50d95)
